code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -234,8 +244,32 @@
 								"carpet tiles" = /obj/item/stack/tile/carpet)
 
 /obj/item/borg/upgrade/service
-	name = "service module board"
-	desc = "Used to give a service cyborg cooking tools."
+	name = "service module upgrade board"
+	desc = "Used to give a service cyborg cooking tools and upgrade their service gripper to be able to handle beakers, food and seeds."
 	icon_state = "cyborg_upgrade2"
 	required_module = list(/obj/item/weapon/robot_module/butler)
-	modules_to_add = list(/obj/item/weapon/reagent_containers/glass/beaker/large/cyborg,/obj/item/weapon/kitchen/utensil/knife/large,/obj/item/weapon/storage/bag/food/borg)
+	modules_to_add = list(/obj/item/weapon/kitchen/utensil/knife/large, /obj/item/weapon/kitchen/rollingpin, /obj/item/weapon/storage/bag/plants)
```

Comment:
Due to the fact that service borgs no longer have built-in trays, and the service borg upgrade no longer contains a food bag, and service borg grippers cannot grip food trays or food bags whether upgraded or not, this means that the ability for service borgs to deliver large amounts of prepared food has been removed entirely.
Why is that?

Reply:
I just realized i added the plant bag instead of the FOOD one. I'll fix it when i get to home a few hours from now.